scrapers/ClubElegantAngel.yml

```diff
@@ -0,0 +1,76 @@
+name: "ClubElegantAngel"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - clubelegantangel.com/video/
+    scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - clubelegantangel.com/dvd/
+    scraper: movieScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+        Name: 
+          fixed: Elegant Angel
+        URL: 
+          fixed: https://www.clubelegantangel.com
+      URL: //head/link[@rel="canonical"]/@href
+      Movies:
+        Name: //div[contains(.,"DVD :")]/a/text()
+        URL: 
+          selector: //div[contains(.,"DVD :")]/a/@href
+          postProcess:
+            - replace:
+                - regex: ^
+                  with: "https://www.clubelegantangel.com"
+      Title: //head/title/text()
+      Date:
+        selector: //div[@class="wrpHighlightUpdated"]/text()
+        postProcess:
+          - parseDate: 2006-01-02 #2012-02-18
+      #Details:
+        #selector: 
+      Performers:
+        Name: //li[contains(.,"Pornstars:")]/a/text()
+        URL:
+          selector: //li[contains(.,"Pornstars:")]/a/@href
+          postProcess:
+            - replace:
+                - regex: ^
+                  with: "https://www.clubelegantangel.com" 
+      Tags:
+        Name: //li[contains(.,"Categories :")]/a/text()
+      Image: //div[@class="gblViewHighlight"]/img/@src
+        #selector: //div[@class="gblViewHighlight"]/img/@src
+        #postProcess:
+          #- replace:
+              #- regex: ^([^?]+).*$
+                #with: "https:$1"
+  movieScraper:
+    movie:
+      Name: //head/title/text()
+      #Director:
+        #selector: 
+        #concat: ", "
+      Duration: //td/div/text()
```

Comment:
The `min` needs to go and we need to append the seconds part
maybe ?
```yaml
      Duration:
        selector: //td/div/text()
        postProcess:
          - replace:
              - regex: (\d+)\s.*
                with: "$1:00"
```

Reply:
i works for me with the min when i scrape the movie i get `2:33` with `153 min`

Reply:
It may work with the min but the end value you will store will be 2:33 not 2:33:00 
All durations are in seconds and thats what we use in all movie scrapers (or should use if we missed a few )

Reply:
a sorry missed that. i changed it and it now works correctly